gradle.properties

```
@@ -2,4 +2,4 @@
 org.gradle.jvmargs=-Xmx4G
 org.gradle.parallel=true
 
-mod_version=5.0.0
+mod_version=6.0.0
```

Major version bumps are only for breaking MC version bumps
```suggestion
mod_version=5.1.0
```

---

done in next commit